app/controllers/api/v1/application_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    class ApplicationController < ActionController::API
+      include DeviseParameter
```

Comment:
I think this one is not needed for API, even `update_allowed_parameters`  is not used in the in any API. Wjat do you think?

Reply:
It makes no sense indeed!! I'm wondering how I could let such a thing here!
Thanks a lot!! :)